public/app/features/alerting/unified/components/silences/SilencesFilter.tsx

```diff
@@ -7,14 +7,15 @@ import { getSilenceFiltersFromUrlParams } from '../../utils/misc';
 import { SilenceState } from 'app/plugins/datasource/alertmanager/types';
 import { parseMatchers } from '../../utils/alertmanager';
 import { debounce } from 'lodash';
+import { v4 } from 'uuid';
```

Comment:
nit: I think the convention is to use lodash's `uniqueId(prefix)` function
ie. `uniqueId('queryString')`

Not a blocker since uuid.v4 is plenty fast